jest.config.json

```diff
@@ -7,5 +7,7 @@
       "statements": 100
     }
   },
-  "transform": {}
+  "transform": {
+    "src.+\\.(t|j)s$": "@swc/jest"
```

Comment:
Explaining this choice of transform: https://blog.joshuakgoldberg.com/jest-babel-to-swc

I don't have a strong preference of `@swc/jest` vs Babel vs `esbuild-jest` vs `ts-jest` vs etc., as long we're not running type checking as part of tests.